src/DIPS.Xamarin.UI/Commands/IAsyncCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace DIPS.Xamarin.UI.Commands
+{
+    /// <summary>
+    /// Aync command without parameters. Converts command executes to task to simply ViewModel creation.
```

Comment:
Not sure what this sentence means. What does it simply? Do you mean simpliefie? I dont think you need to add the last part.